daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1230,6 +1350,17 @@ func (devices *DeviceSet) initDevmapper(doInit bool) error {
 		}
 	}
 
+	// Pool already exists and caller did not pass us a pool. That means
+	// we probably created pool earlier and could not remove it as some
+	// containers were still using it. Detect some of the properties of
+	// pool, like is it using loop devices.
+	if info.Exists != 0 && devices.thinPoolDevice == "" {
```

Comment:
is there no reason to get the major and minor info when `devices.thinPoolDevice` is already set?